src/OrbitGl/DataView.cpp

```diff
@@ -4,12 +4,27 @@
 
 #include "DataView.h"
 
+#include <absl/strings/str_join.h>
+#include <absl/strings/str_split.h>
+
 #include <memory>
 
 #include "App.h"
 #include "OrbitBase/File.h"
 #include "OrbitBase/Logging.h"
 
+namespace orbit_gl {
+std::string FormatValueForCsv(std::string_view value) {
+  std::string result;
+  result.append("\"");
+  std::vector<std::string> split_value = absl::StrSplit(value, "\"");
+  std::string value_with_escaped_quotes = absl::StrJoin(split_value, "\"\"");
```

Comment:
Maybe `absl::StrReplaceAll({"\"", "\"\""})`?

Reply:
Great, that's better, will change it!